src/commands/web/index.ts

```diff
@@ -28,7 +28,8 @@ const exampleStreamConfig: StreamConfig = {
   streamWeb: true,
   streamWebFolder: '/example/folder/path',
   assetPaths: [],
-  webSourcePath: '/example/path'
+  webSourcePath: '/example/path',
+  streamServiceName: 'clickme'
 }
 
 export async function createWebAppServices(targetName: string) {
```

Comment:
Why not passing Target as passing to `compile` `build` `deploy`,

Reply:
Good point. Fixing this now.

Reply:
Fixed.